src/komponenter/header/header-regular/common/varsler/varselvisning/Varselvisning.tsx

```diff
@@ -28,17 +28,14 @@ export const Varselvisning = ({ setKbId }: Props) => {
         <div className={isTomListe ? 'varsler-visning-tom' : 'varsler-visning'}>
```

Comment:
Veldig flisespikkeri, men kan evt gjøre noe sånt slik at den bare sjekker en gang om varsler er definert 😄 
```suggestion
        const antallVarsler = varsler ? varsler.oppgaver.length + varsler.beskjeder.length : 0;
        const isTomListe = antallVarsler === 0;
```